pandas/tests/scalar/timestamp/test_arithmetic.py

```diff
@@ -109,6 +110,37 @@ def test_timestamp_sub_datetime(self):
         assert (ts - dt).days == 1
         assert (dt - ts).days == -1
 
+    def test_subtract_tzaware_datetime(self):
+        t1 = Timestamp("2020-10-22T22:00:00+00:00")
+        t2 = datetime(2020, 10, 22, 22, tzinfo=timezone.utc)
+
+        result = t1 - t2
+
+        assert isinstance(result, Timedelta)
+        assert result == Timedelta("0 days")
+
+    def test_subtract_timestamp_from_different_timezone(self):
+        t1 = Timestamp("20130101").tz_localize("US/Eastern")
+        t2 = Timestamp("20130101").tz_localize("CET")
+
+        result = t1 - t2
+
+        assert isinstance(result, Timedelta)
+        assert result == Timedelta("0 days 06:00:00")
+
+    def test_subtracting_involving_datetime_with_different_tz(self):
```

Comment:
could use tz_aware_fixture

Reply:
it seems like we need to modify the fixture to pair an expectation for the offset, if we want to do this. Wondering what concern you want to address with this? It seems like there are two different things to potentiallly cover:

1. varying the format in which the offset is specified (e.g. `US/Eastern`, `Fixedoffset(-250)`, `timezone(timedelta(hours-5))`) to assert all result in the same offset
2. varying the timezone to make sure we can compute the offset implied (e.g. "US/Eastern", "US/Pacific", "Asia/Tokyo")

Reply:
I added a fixture for capturing the expected time difference, and wrote a test to assert we can compute the right value